# Translations template for ironic.
# Copyright (C) 2014 ORGANIZATION
# This file is distributed under the same license as the ironic project.
#
# Translators:
# Andi Chandler <andi@gowling.com>, 2014
msgid ""
msgstr ""
"Project-Id-Version: Ironic\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2014-08-28 06:10+0000\n"
"PO-Revision-Date: 2014-08-11 14:21+0000\n"
"Last-Translator: Andi Chandler <andi@gowling.com>\n"
"Language-Team: English (United Kingdom) (http://www.transifex.com/projects/p/"
"ironic/language/en_GB/)\n"
"Language: en_GB\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 1.3\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: ironic/common/images.py:133
#, python-format
msgid "vfat image creation failed. Error: %s"
msgstr ""

#: ironic/common/images.py:201
msgid "Creating the filesystem root failed."
msgstr ""

#: ironic/common/images.py:215
msgid "Creating ISO image failed."
msgstr ""

#: ironic/drivers/modules/agent.py:388
#, python-format
msgid "Async exception for %(node)s: %(msg)s"
msgstr "Async exception for %(node)s: %(msg)s"

#: ironic/drivers/modules/ipminative.py:348
#, python-format
msgid ""
"IPMI set boot device failed for node %(node_id)s with the following error: "
"%(error)s"
msgstr ""
"IPMI set boot device failed for node %(node_id)s with the following error: "
"%(error)s"

#: ironic/drivers/modules/ipminative.py:383
#, python-format
msgid ""
"IPMI get boot device failed for node %(node_id)s with the following error: "
"%(error)s"
msgstr ""
"IPMI get boot device failed for node %(node_id)s with the following error: "
"%(error)s"

#: ironic/drivers/modules/iscsi_deploy.py:294
#, python-format
msgid "Error returned from deploy ramdisk: %s"
msgstr ""

#: ironic/drivers/modules/iscsi_deploy.py:307
#: ironic/drivers/modules/pxe.py:430
#, python-format
msgid "Deploy failed for instance %(instance)s. Error: %(error)s"
msgstr ""

#: ironic/drivers/modules/pxe.py:408
#, python-format
msgid "Node %s is not waiting to be deployed."
msgstr ""

#: ironic/drivers/modules/seamicro.py:550
#, python-format
msgid ""
"Seamicro set boot device failed for node %(node)s with the following error: "
"%(error)s"
msgstr ""
"Seamicro set boot device failed for node %(node)s with the following error: "
"%(error)s"

#: ironic/drivers/modules/ssh.py:643
#, python-format
msgid ""
"Failed to set boot device for node %(node)s, virt_type %(vtype)s does not "
"support this operation"
msgstr ""
"Failed to set boot device for node %(node)s, virt_type %(vtype)s does not "
"support this operation"

#: ironic/drivers/modules/drac/power.py:65
#, python-format
msgid ""
"DRAC driver failed to get power state for node %(node_uuid)s. Reason: "
"%(error)s."
msgstr ""

#: ironic/drivers/modules/drac/power.py:96
#: ironic/drivers/modules/drac/power.py:108
#, python-format
msgid ""
"DRAC driver failed to set power state for node %(node_uuid)s to "
"%(target_power_state)s. Reason: %(error)s."
msgstr ""

#: ironic/nova/virt/ironic/driver.py:297
#, python-format
msgid ""
"Failed to clean up the parameters on node %(node)s when unprovisioning the "
"instance %(instance)s"
msgstr ""
"Failed to clean up the parameters on node %(node)s when unprovisioning the "
"instance %(instance)s"

#: ironic/nova/virt/ironic/driver.py:606
#, python-format
msgid ""
"Error preparing deploy for instance %(instance)s on baremetal node %(node)s."
msgstr ""
"Error preparing deploy for instance %(instance)s on baremetal node %(node)s."

#: ironic/nova/virt/ironic/driver.py:631
#, python-format
msgid "Error deploying instance %(instance)s on baremetal node %(node)s."
msgstr "Error deploying instance %(instance)s on baremetal node %(node)s."

#: ironic/openstack/common/excutils.py:76
#, python-format
msgid "Original exception being dropped: %s"
msgstr "Original exception being dropped: %s"

#: ironic/openstack/common/excutils.py:105
#, python-format
msgid "Unexpected exception occurred %d time(s)... retrying."
msgstr "Unexpected exception occurred %d time(s)... retrying."

#: ironic/openstack/common/lockutils.py:119
#, python-format
msgid "Could not release the acquired lock `%s`"
msgstr "Could not release the acquired lock `%s`"

#: ironic/openstack/common/loopingcall.py:95
msgid "in fixed duration looping call"
msgstr "in fixed duration looping call"

#: ironic/openstack/common/loopingcall.py:138
msgid "in dynamic looping call"
msgstr "in dynamic looping call"

#: ironic/openstack/common/periodic_task.py:202
#, python-format
msgid "Error during %(full_task_name)s: %(e)s"
msgstr "Error during %(full_task_name)s: %(e)s"

#: ironic/openstack/common/service.py:188
msgid "Exception during rpc cleanup."
msgstr "Exception during rpc cleanup."

#: ironic/openstack/common/service.py:277
msgid "Unhandled exception"
msgstr "Unhandled exception"
